f90wrap/parser.py

```diff
@@ -46,7 +46,7 @@
 import os
 import re
 
-from f90wrap.fortran import (Fortran, Root, Program, Module,
+from f90wrap.fortran import (Fortran, Binding,Root, Program, Module,
```

Comment:
Minor nit-picking comment: please could you include a space after the `,` and move Binding to the end of the import list, since the ordering is from low- to high-level classes.

Reply:
I followed your suggestion and pushed the changes.